tests/x11/nis_client.pm

```diff
@@ -14,80 +14,109 @@
+    type_string $setup_nis_nfs_x11{nfs_domain};
+    send_key 'alt-f';        # open firewall port
+    assert_screen 'nis-client-nfs-settings-tab-opened-fw';
+    wait_still_screen 4, 4;    # blinking cursor
     save_screenshot;
-    send_key 'alt-n';                       # nfs shares tab
-    send_key 'alt-a';                       # add
+}
+
+sub nfs_shares_tab {
+    send_key 'alt-n';          # nfs shares tab
+    assert_screen 'nis-client-nfs-client-shares-conf';
+    send_key 'alt-a';          # add
     wait_still_screen 4;
-    send_key 'alt-v';                       # NFSV4 share checkbox
-    send_key 'alt-s';                       # choose NFS server button
+    send_key 'alt-v';          # NFSV4 share checkbox
+    send_key 'alt-s';          # choose NFS server button
     assert_screen 'nis-client-nfs-server';
-    send_key 'alt-o';                       # OK
-    send_key 'alt-r';                       # remote directory text field
-    type_string '/home/nis_user';
-    send_key 'alt-m';                       # mount point text field
-    type_string '/home/nis_user';
-    send_key 'alt-o';                       # OK
+    send_key 'alt-o';          # OK
+    send_key 'alt-r';          # remote directory text field
+    type_string $setup_nis_nfs_x11{nfs_dir};
+    send_key 'alt-m';          # mount point text field
+    type_string $setup_nis_nfs_x11{nfs_dir};
+    send_key 'alt-o';          # OK
     assert_screen 'nis-client-nfs-client-configuration';
-    send_key 'alt-o';                       # OK
-    assert_screen 'nis-client-configuration';
-    send_key 'alt-f';                       # finish
+    send_key 'alt-o';          # OK
+    assert_screen 'nis-client-fw-opened';
+    send_key 'alt-f';          # finish
+}
+
+sub setup_verification {
     assert_screen 'yast2_closed_xterm_visible', 90;
-    script_run 'mount|grep nfs';            # print nfs mounts
+    script_run 'mount|grep nfs';    # print nfs mounts
     if (is_sle('>=15')) {
         record_soft_failure('bsc#1090886');
-        script_run('mount -vt nfs /home/nis_user');
+        script_run("mount -vt nfs $setup_nis_nfs_x11{nfs_dir}");
+        script_run 'mount|grep nfs';    # verify mounted nfs
+    }
+    (my $server_ip = $setup_nis_nfs_x11{server_address}) =~ m/\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}/;
+    script_run "cat /proc/mounts | grep -i $server_ip:";
+    # create file with text, will be checked by server
+    script_run 'echo ' . $setup_nis_nfs_x11{message} . " > " . $setup_nis_nfs_x11{nfs_dir} . '/test';
+}
+
+sub run {
+    my ($self) = @_;
+    x11_start_program('xterm -geometry 155x45+5+5', target_match => 'xterm');
+    turn_off_gnome_screensaver if check_var('DESKTOP', 'gnome');
+    become_root;
+    setup_static_mm_network($setup_nis_nfs_x11{client_address});
+    assert_script_run 'zypper -n in yast2-nis-server';
+
+    if ($self->firewall eq 'firewalld') {
+        record_soft_failure('bsc#1089851');
```

Comment:
This bug was marked as duplicate of https://bugzilla.suse.com/show_bug.cgi?id=1083487 , could you please update bug ref?